apps/artemis-web/components/401-hoc/401-hoc.tsx

```diff
@@ -20,7 +20,7 @@ const AuthHOC = (WrappedComponent, ACL = []) => {
       router.push('/login');
     }
 
-    if (user && !loading && !ACL.includes(user.role)) {
+    if (user && !loading && (!ACL.includes(user.role) || blockUserType === user.type)) {
```

Comment:
what does this do here exactly? What changes among ldap and normal user for the 401-hoc?

Reply:
It says if the user type is equal with the blockUserType, then block access.

The blockUserType is being passed by the specific page. e.g. if I want the system page to be unaccessible by ldap users, I can do that.

Reply:
I see, OK fine as-is then, good adjustment